module/observations/helper.js

```diff
@@ -1048,4 +1050,46 @@ module.exports = class ObservationsHelper {
         });
     }
 
+
+         /**
+      * observation link.
+      * @method
+      * @name getObservationLink
+      * @param  {String} observationSolutionId observation solution external Id.
+      * @param  {String} appName name of app.
+      * @returns {getObservationLink} observation getObservationLink.
+     */
+
+    static getObservationLink(observationSolutionId, appName) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let observationData = await database.models.solutions.findOne({
+                    externalId: observationSolutionId,
+                    isReusable: false,
+                    type : messageConstants.common.OBSERVATION
+                }).lean();
+
+                let appData = await database.models.apps.findOne({
```

Comment:
@priyanka-TL  apps model is not there in samiksha . Its functionality and models is there in kendra . So call kendra apis please.